docs/element-definitions.md

```diff
@@ -1232,6 +1250,12 @@ Note: At the moment only the HTTP protocol is supported.
         For the time being this reference document defines only HTTP-specific transaction
         data structures.
 
+#### Classifications
+
+- `"host"` - A host transition represents the "root" of the API transition. 
+    The transition href MAY be append to the host href to create a absolute URI.
+    A transition that has a `host` classification MUST be a root component of a URI.
+
```

Comment:
A transition cannot be a host no? Just a resource. I think we can remove this paragraph.

Reply:
Being Application State Transitions (also called "Transitions") resource operations should we include the Category of Hosts at all (https://github.com/apiaryio/api-elements/issues/53#issuecomment-602679998)?

Reply:
removed